universum/modules/vcs/perforce_vcs.py

```diff
@@ -199,6 +199,7 @@ def submit_new_change(self, description, file_list, review=False, edit_only=Fals
         change["Files"] = []
         change["Description"] = description
         change_id = self.p4.save_change(change)[0].split()[1]
+        self.out.log(f"Created empty change {change_id} for reconciling.")
```

Comment:
Is "change" the correct term used by the perforce?

Reply:
I prefer "CL"

Reply:
I think I've seen term 'change' somewhere in official p4 docs, and therefore tend to use them as synonyms. But 'CL' is definitely more formal.